doc/managing-wallets.md

```diff
@@ -108,7 +108,11 @@ Wallets created before version 0.13 are not HD and must be backed up every 100 k
 
 ### 1.6 Restoring the Wallet From a Backup
 
-To restore a wallet, the `restorewallet` RPC must be used.
+To restore a wallet, the `restorewallet` RPC or the `Restore Wallet` GUI menu item must be used.
+
+In the GUI, a backup can be restored via `File` -> `Restore Wallet...`.
+
+Alternatively, a backup can be restored using the `restorewallet` RPC.
```

Comment:
I agree with this. 
I believe that the reason that it is repeated here is because it needs to refer to the RPC before the command in the line below. An alternative could be to follow the same approach as the "1.4 Backing Up the Wallet" section and give a bit more direction/information on the arguments of the RPC commands. But removing it also makes sense, if the above GUI reference is moved.